synapse/tests/test_tools_cryo_cat.py

```diff
@@ -1,8 +1,14 @@
+import io
+from unittest.mock import Mock
+
+import msgpack
+
 import synapse.cryotank as s_cryotank
 
 import synapse.tools.cryo.cat as s_cryocat
 
 from synapse.tests.common import *
+from synapse.lib.iq import redirectStdin
```

Comment:
when this guy becomes part of syntest we can remove this import

Reply:
ok